user-interface/modals/content-library-modal.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable padded-blocks */
+const { getActivityCollection } = require('../../services/strapi/index');
+
+const baseUrl = process.env.STRAPI_BASE_URL;
+
+// eslint-disable-next-line consistent-return
+async function contentLibraryModal(body) {
+  const collections = await getActivityCollection();
```

Comment:
Why are we fetching all the collections and finding a specific collection when we have access to the collection id. 
We should fetch only a single collection.